apps/meteor/app/livechat/server/lib/RoutingManager.ts

```diff
@@ -65,47 +65,27 @@ export const RoutingManager: Routing = {
 	methodName: null,
 	methods: {},
 
-	startQueue() {
-		// todo: move to eventemitter or middleware
-		// queue shouldn't start on CE
-	},
-
-	isMethodSet() {
-		return !!this.methodName;
-	},
-
-	async setMethodNameAndStartQueue(name) {
-		logger.info(`Changing default routing method from ${this.methodName} to ${name}`);
-		if (!this.methods[name]) {
-			logger.warn(`Cannot change routing method to ${name}. Selected Routing method does not exists. Defaulting to Manual_Selection`);
-			this.methodName = 'Manual_Selection';
-		} else {
-			this.methodName = name;
-		}
-
+	async startQueue() {
 		const shouldPreventQueueStart = await License.shouldPreventAction('monthlyActiveContacts');
 
 		if (shouldPreventQueueStart) {
 			logger.error('Monthly Active Contacts limit reached. Queue will not start');
 			return;
 		}
-
 		void (await Omnichannel.getQueueWorker()).shouldStart();
 	},
 
+	isMethodSet() {
+		return settings.get<string>('Livechat_Routing_Method') !== '';
+	},
+
 	// eslint-disable-next-line @typescript-eslint/naming-convention
 	registerMethod(name, Method) {
 		this.methods[name] = new Method();
 	},
 
 	getMethod() {
-		if (!this.methodName) {
-			throw new Meteor.Error('error-routing-method-not-set');
-		}
-		if (!this.methods[this.methodName]) {
-			throw new Meteor.Error('error-routing-method-not-available');
-		}
-		return this.methods[this.methodName];
+		return this.methods[settings.get<string>('Livechat_Routing_Method')];
```

Comment:
I believe we should keep the validation here, what if for some reason the selected method is not registered here?